lib/file_transfer_agent/file_transfer_agent.js

```diff
@@ -600,7 +601,14 @@ function FileTransferAgent(context) {
       const src = data['src_locations'][0];
 
       // Get root directory of file path
-      const root = path.dirname(src);
+      let root = path.dirname(src);
+
+      // If root is relative . then replace with context's cwd
+      // Used for VS Code extension where extension cwd differs from user workspace dir
+      if (root === '.') {
```

Comment:
I think I have to replace the relative with the cwd. This check isn't sufficient since if the user is trying to PUT something relative into a folder within the cwd it doesn't cover it. 
ex:
put file://./wow/test.csv @~ overwrite=True;

Also need to fix this for GET commands.